README.adoc

```diff
@@ -64,6 +64,7 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+For a new pull request, the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered. However, the build job runs the integration tests only when the pull request contains any change related to add-on.
```

Comment:
Any changes related to the feature file of the addon, we should be clear about this, cause AFAIU if and old addon is changed and no changes done to the feature file the PR job won't run the integration tests?